### DISCUSSION TITLE
Change color and icon folder

### DISCUSSION THREAD

**Discussion by ghost** [community/community#30989] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi all!

I would like to know if it is possible to change the color of my repository folders and icons, I would like to have an icon and folder customization in my repos in GitHub.

**Comment by radar-blip** [comment]
I just looked through settings, I dont believe that is possible at the moment.

**Comment by theglobehead** [comment]
There is no such feature yet.

You can try adding custom css to your browser.
If you're using chrome, you can use [this plugin]( https://chrome.google.com/webstore/detail/stylish-custom-themes-for/fjnbnpbmkenffdnngjfgmeleoegfcffe?hl=en)

You could change the color of the SVGs using the the filter attribute. 
You can use [this codepen site](https://codepen.io/sosuke/pen/Pjoqqp), to determine the needed filters.

I'm not sure how to change the svg itself. Perhaps you could try using the :after selector

Hope this helps!

**Reply by ghost** [reply]
@theglobehead Hey there! I liked your answer, but it didn't help my problem :( . I wanted to customize my icons and folders. I wanted this feature to be something native, without any extension to work. But... thanks for the help anyway.

**Comment by melvincwng** [comment]
@codehangen Hey there, as others have already mentioned, there is no such functionality that's available currently to change the color & icons of the folders in your GitHub repository.

With that being said, if you really do want such a feature to be implemented, perhaps you can create a feature request post on the forums.

If it gains enough traction and support, perhaps GitHub might end up implementing this as a feature in the future. Who knows? Haha

Otherwise, the only option now is to stick with the default icons GitHub provides for the folders.

Hope this addresses your concerns, thanks 🙏

**Reply by ghost** [reply]
@melvincwng Hi! 
> With that being said, if you really do want such a feature to be implemented, perhaps you can create a feature request post on the forums.
 - What I'm trying to request is this.

**Comment by HypertextAssassinRajith** [comment]
I also lie to change the color of my repo

**Comment by jfullstackdev** [comment, marked as answer]
there is the online editor for GitHub that makes the folders native, but I don't know whether you can further customize it by installing plugins because not all plugins can be installed at that online editor, at least it changed the icons with VS Code look 

![image](https://user-images.githubusercontent.com/47092464/187207288-8302868e-4e7e-4274-8ea0-94a2426bda31.png)

just simply press the dot key on your keyboard or change the URL of the repo, `github.com` to `github.dev`

**Reply by ghost** [reply]
Hi jdevstatic! Can I have all Vscode desktop features in github.dev? (yes or no)?

**Reply by jfullstackdev** [reply]
nope, even in Codespaces, not all can be installed

**Reply by ghost** [reply]
jdevstatic I realized seeing this: 
> "The 'vscode-icons' extension is not available in Visual Studio Code for the Web." and I see this:

![image](https://user-images.githubusercontent.com/111701513/187213929-c6bcc49c-49c6-403e-b13d-ae79395e38fd.png)

I read this: [code.visualstudio.com/blogs/2016/09/08/icon-themes](https://code.visualstudio.com/blogs/2016/09/08/icon-themes) , [stackoverflow.com/questions/66111153/vs-code-material-icon-not-rendering-icon-for-certain-folders](https://stackoverflow.com/questions/66111153/vs-code-material-icon-not-rendering-icon-for-certain-folders) , [code.visualstudio.com/blogs/2016/09/08/icon-themes](https://code.visualstudio.com/blogs/2016/09/08/icon-themes) but it shows the same error. ***So... what to do was install *material-icon-theme* instead of *vscode-icons*. This solved the previous problem I reported, but one last question arose..***

**Reply by ghost** [reply]
jdevstatic This is my last question, if you can help me.  Is it possible to sync the github.dev settings with the vscode I installed on my desktop? *it is possible? (yes or no)*

#### 
![image](https://user-images.githubusercontent.com/111701513/187213278-5ad386dd-644c-4d99-b565-06135837b93a.png)

**Reply by jfullstackdev** [reply]
I'm not very sure whether the online editor `.dev` includes that capability but the Codespaces can, if you have it, but anyways you can try that syncing

https://docs.github.com/en/codespaces/customizing-your-codespace/personalizing-github-codespaces-for-your-account

**Comment by ghost** [comment]
@Hypertext-Assassin-RSS @linuxlawson @theglobehead @melvincwng @jdevstatic One more thing, I would like to thank you all for everything. Thank you all so much for the feedback. You all are amazing.
